src/lib/packager/product_patterns.rb

```diff
@@ -0,0 +1,141 @@
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+#
+
+require "yast"
+
+module Packager
```

Comment:
Well, convention is not set in stone. My personal convention is that something YaST specific go to Yast namespace and something that is more independent is more likely should go to own namespace, so we can in future separate it if needed.